phpstan-baseline.neon

```diff
@@ -566,6 +566,11 @@ parameters:
 			count: 1
 			path: src/Rules/Generics/TemplateTypeCheck.php
 
+		-
+			message: "#^Method PHPStan\\\\Rules\\\\Keywords\\\\RequireFileExistsRule\\:\\:processNode\\(\\) should return list\\<PHPStan\\\\Rules\\\\IdentifierRuleError\\> but returns array\\{PHPStan\\\\Rules\\\\RuleError\\}\\.$#"
+			count: 1
+			path: src/Rules/Keywords/RequireFileExistsRule.php
```

Comment:
This can't be baselined. The error needs an identifier.

Reply:
Fixed in https://github.com/phpstan/phpstan-src/pull/3294/commits/5d4563383f16601d47175086e8b154a4f43efc91